config/rbac/contour_viewer_role.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: contour-viewer-role
+  name: contour-viewer
```

Comment:
Why not `contour-operator-viewer`?

Reply:
@jpeach b/c the role is to view the `Contour` custom resource. The operator's service account will be given this cluster role so it can view `Contour` custom resources.